flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/util/HadoopConfigLoader.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.flink.fs.s3.common;
+package org.apache.flink.runtime.util;
```

Comment:
I'm wondering whether we shouldn't put the `HadoopConfigLoader` into the `flink-fs-hadoop-shaded` module, because it is intended for self-contained fs implementations which don't read a HDFS/Yarn configuration from disk but simply instantiate the `Configuration` contained in this module.